EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -449,36 +450,41 @@ partial class Program {
             return commandParameters;
         }
 
-        public List<Token> ParseTokens(String commandString) => (String.IsNullOrWhiteSpace(commandString) || commandString.Trim().StartsWith("#")) ? NewList<Token>() :
-            ParseSurrounded(commandString, "`\'\"",
+        delegate IEnumerable<String> Pass(String s);
+        public List<Token> Tokenize(String commandString) {
+            Pass thirdPass = v => SeperatorPass(v, thirdPassTokens);
+            Pass secondPass = v => SeperatorPass(v, secondPassTokens, w => PrimitivePass(w, thirdPass));
+            Pass firstPass = v => SeperatorPass(v, firstPassTokens, w => PrimitivePass(w, secondPass));
```

Comment:
Perhaps we can removes the double primitive pass once we get the vector rule in place :)